site/en/install/lang_java.md

```diff
@@ -1,207 +1,203 @@
-  <groupId>org.myorg</groupId>
-  <artifactId>hellotensorflow</artifactId>
-  <version>1.0-SNAPSHOT</version>
-  <properties>
-    <exec.mainClass>HelloTensorFlow</exec.mainClass>
-	<!-- The sample code requires at least JDK 1.7. -->
-	<!-- The maven compiler plugin defaults to a lower version -->
-	<maven.compiler.source>1.7</maven.compiler.source>
-	<maven.compiler.target>1.7</maven.compiler.target>
-  </properties>
-  <dependencies>
-    <dependency>
-	  <groupId>org.tensorflow</groupId>
-	  <artifactId>tensorflow</artifactId>
-	  <version>1.14.0</version>
-	</dependency>
-  </dependencies>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.myorg</groupId>
+    <artifactId>hellotensorflow</artifactId>
+    <version>1.0-SNAPSHOT</version>
+	
+    <properties>
+	<!-- Minimal version for compiling TensorFlow Java is JDK 8 -->
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+    </properties>
+	
+    <dependencies>
+	<!-- Includes vanilla TensorFlow for all platforms -->
+        <dependency>
+            <groupId>org.tensorflow</groupId>
+            <artifactId>tensorflow-core-platform</artifactId>
+            <version>0.2.0</version>
+        </dependency>
+    </dependencies>
 </project>
 ```
 
-Create the source file (`src/main/java/HelloTensorFlow.java`):
+Create the source file `src/main/java/HelloTensorFlow.java`:
 
 ```java
-import org.tensorflow.Graph;
-import org.tensorflow.Session;
+import org.tensorflow.ConcreteFunction;
+import org.tensorflow.Signature;
 import org.tensorflow.Tensor;
-import org.tensorflow.TensorFlow;
+import org.tensorflow.op.Ops;
+import org.tensorflow.op.core.Placeholder;
+import org.tensorflow.op.math.Add;
+import org.tensorflow.types.TInt32;
 
 public class HelloTensorFlow {
+
   public static void main(String[] args) throws Exception {
-	try (Graph g = new Graph()) {
-	  final String value = "Hello from " + TensorFlow.version();
-
-	  // Construct the computation graph with a single operation, a constant
-	  // named "MyConst" with a value "value".
-	  try (Tensor t = Tensor.create(value.getBytes("UTF-8"))) {
-	    // The Java API doesn't yet include convenience functions for adding operations.
-		g.opBuilder("Const", "MyConst").setAttr("dtype", t.dataType()).setAttr("value", t).build();
-	  }
-
-	  // Execute the "MyConst" operation in a Session.
-	  try (Session s = new Session(g);
-	      // Generally, there may be multiple output tensors,
-		  // all of them must be closed to prevent resource leaks.
-		  Tensor output = s.runner().fetch("MyConst").run().get(0)) {
-	    System.out.println(new String(output.bytesValue(), "UTF-8"));
-	  }
+    try (ConcreteFunction dbl = ConcreteFunction.create(HelloTensorFlow::dbl);
+        Tensor<TInt32> x = TInt32.scalarOf(10);
+        Tensor<TInt32> dblX = dbl.call(x).expect(TInt32.DTYPE)) {
+      System.out.println("Double of " + x.data().getInt() + " is " + dblX.data().getInt());
```

Comment:
`System.out.println(x.data().getInt() + " doubled is " + dblX.data().getInt());`